storage.rules

```diff
@@ -2,11 +2,13 @@ rules_version = '2';
 service firebase.storage {
   match /b/{bucket}/o {
     match /assets/{allPaths=**} {
-        allow read
+        allow read;
     }
     match /users/{uid}/{allPaths=**} {
-        allow read
-        allow write, update: if request.auth.uid == uid
+        allow read;
+        // Allow users to create new files under their unique path.
+        // Don't allow updating files so we can cache files long-term
+        allow write: if request.auth.uid == uid && resource == null;
```

Comment:
will this effect the attemp to update the profile pic?

Reply:
Yes. It'll block it. This is also taken care of by the uuid, so it's not strictly necessary but I think it's good to block it here as well so the Firebase raises a permission denied error.

Reply:
I guess it's a really edge case. Can we just leave the `auth.uid == uid` check here?